utils/downloads.py

```diff
@@ -38,26 +33,10 @@ def download_sounds(sounds_list, licenses_url, pack):
     the wav files of the sonds and a text file with the license. This response
     is handled by mod_zipfile of nginx to generate a zip file with the content.
     """
-    users = User.objects.filter(sounds__in=sounds_list).distinct()
-    # Generate text file with license info
-    license_ids = sounds_list.values('license_id').distinct()
-    licenses = License.objects.filter(id__in=license_ids).all()
-    attribution = render_to_string("sounds/pack_attribution.txt",
-        dict(users=users,
-            pack=pack,
-            licenses=licenses,
-            sound_list=sounds_list))
-
-    tmpf = tempfile.NamedTemporaryFile(dir=settings.PACKS_PATH, delete=False)
-    tmpf.write(attribution.encode("UTF-8"))
-    tmpf.close()
-    secret_name = tmpf.name.replace(settings.PACKS_PATH, settings.PACKS_SENDFILE_URL)
-
-    os.chmod(tmpf.name, stat.S_IWUSR | stat.S_IRUSR | stat.S_IRGRP | stat.S_IROTH)
+    attribution = pack.get_attribution()
```

Comment:
no pack argument.

It'd be good to see some tests for these methods to catch these errors!